hci_ninafw.go

```diff
@@ -151,6 +162,12 @@ func (h *hci) reset() error {
 }
 
 func (h *hci) poll() error {
+	if h.softRTS != machine.NoPin {
+		h.softRTS.Low()
+
+		defer h.softRTS.High()
+	}
```

Comment:
I just hope there won't be a buffer overrun. There is a chance this could happen with a GC cycle at the wrong time, for example.

If you want to make things a bit more robust, you could toggle RTS on each byte read, like so:

```
loop:
    enable RTS
    while buffered == 0:
        continue
    read byte
    disable RTS
    process byte
```
(It's probably a bit slower though)

Reply:
I tried that and it was definitely slower. I will leave as it is for now and consider some solution for a future PR.

Thanks for review @aykevl now merging.